### ISSUE TITLE
l2d has added to the new version! how about spine!?

### ISSUE THREAD

**Issue by RP1992N** [Vadim07/AssetStudio#36]
i hope there is feature to detect spine assets and export it without manually selecting each of require files.
thanks!

**Comment by aelurum** [comment]
Yeah, pretty logical assumption.
A spine exporter was also planned along with the l2d exporter, but was postponed.
Though it's still on my TODO list.

**Comment by Yumeo0** [comment]
Has there been any progress on this so far or is it still being postponed?

**Comment by aelurum** [comment]
No progress so far.
This is not really a high priority feature since you can easily export spines manually.

**Comment by Yumeo0** [comment]
Really? If I try to import the files I exported into the Spine software it doesn't load anything correctly. Do I need to import them into the unity spine runtime for them to work?

**Comment by aelurum** [comment]
Well, then you need to do some additional manipulation with your spine models and also make sure that your Spine software supports the version of your spine models.
It is unlikely that this is related to AssetStudio itself.
In most cases it is enough to simply export the spine assets (*.skel, *.atlas, *.png).

**Comment by RP1992N** [comment]
Yeah! I have do it manually using asset studio for bunch of files. The difference is i need to change the name of the extension to match the spine type of model like skel, json, and atlas. Just write some python script to do it automatically. But what i want to do in this asset studio is to automatically filter the spine type and export automatically. I think some spine is often called in Monobehavior *_SkeletonData (skel, texture, and atlas)
Maybe you can do something about it. If you have free time ofc.

**Comment by Yumeo0** [comment]
> Well, then you need to do some additional manipulation with your spine models and also make sure that your Spine software supports the version of your spine models. It is unlikely that this is related to AssetStudio itself. In most cases it is enough to simply export the spine assets (*.skel, *.atlas, *.png).

I tried another file and it seems like the first one was corrupted. Every other one I tried works. I feel so dumb for not thinking about that before 😭

**Comment by Yumeo0** [comment]
> Yeah! I have do it manually using asset studio for bunch of files. The difference is i need to change the name of the extension to match the spine type of model like skel, json, and atlas. Just write some python script to do it automatically. But what i want to do in this asset studio is to automatically filter the spine type and export automatically. I think some spine is often called in Monobehavior *_SkeletonData (skel, texture, and atlas) Maybe you can do something about it. If you have free time ofc.

You only have to add a .txt to the .skel and .atlas if you want to load them in the unity runtime.
